frontend/src/pages/library/LibraryPage.tsx

```diff
@@ -1,6 +1,120 @@
-import React, { FunctionComponent } from "react";
+import React, { FunctionComponent, useState } from "react";
+import { useRecoilValue } from "recoil";
+import cx from "classnames";
+import { Table } from "../../components/Table/Table";
+import "./LibraryPage.scss";
+import { methodologiesAtom, projectsAtom } from "../../store/atoms";
+import { SuspenseSpinner } from "../../components/SuspsenseSpinner";
 import PageLayout from "../../components/PageLayout/PageLayout";
+import { Button } from "../../components/Button/Button";
+import PlusSvg from "../../assets/svg/plus.svg";
 
 export const LibraryPage: FunctionComponent = () => {
-  return <PageLayout>Library Page</PageLayout>;
+  return (
+    <PageLayout>
+      <EntitiesTab
```

Comment:
Looks like over engineering but I do not want to argue. Just remove copy paste

Reply:
>over engineering

>remove copy paste

it was introduced exactly to reduce the copy-paste. this copy-paste is eliminated in further PRs